docs/_docs/templates.md

```diff
@@ -416,9 +416,14 @@ The default is `default`. They are as follows (with what they filter):
 - `default`: spaces and non-alphanumeric characters
 - `pretty`: spaces and non-alphanumeric characters except for `._~!$&'()+,;=@`
 
-## Tags
+## Jekyll Tags
 
-### Includes
+* [Includes](#include)
+* [Code snippet highlighting](#highlight)
+* [Linking to pages, collections and posts (the new and improved way)](#link)
+* [Linking to posts (the old way)](#post_url)
+
+### Includes {#include}
```

Comment:
and changes these back, and I'm not sure I'd like to introduce new anchors, so I'm getting rid of any I introduced.